spec/models/parent_object_spec.rb

```diff
@@ -3,5 +3,63 @@
 require 'rails_helper'
 
 RSpec.describe ParentObject, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before do
+    prep_metadata_call
+  end
+  context "a newly created ParentObject with just the oid and default authoritative_metadata_source (Ladybird for now)" do
+    let(:po) { described_class.create(oid: "2004628") }
+    before do
+      stub_request(:get, "https://metadata-api-test.library.yale.edu/metadatacloud/api/ladybird/oid/2004628")
+        .to_return(status: 200, body: File.open(File.join(fixture_path, "ladybird", "2004628.json")).read)
+      po
```

Comment:
The stubbing is cool - you should walk through this for everyone.

I'm not sure I understand the value of separating the let and the call to `po` instead of just having
`po = described_class.create(oid: "2004628")` in the before block.  I.E. a line that just says `po` is pretty opaque - at least maybe call it `parent_object` there's really no tax on extra keystrokes and they make the code much more legible in the future.